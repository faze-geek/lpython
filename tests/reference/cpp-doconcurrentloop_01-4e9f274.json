{
    "basename": "cpp-doconcurrentloop_01-4e9f274",
    "cmd": "lpython --no-color --show-cpp {infile}",
    "infile": "tests/doconcurrentloop_01.py",
    "infile_hash": "dc0022671a9fa7b620378903a700dd8c4390297cacb1aca2b904069d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "cpp-doconcurrentloop_01-4e9f274.stderr",
    "stderr_hash": "e95e3a307bd47fd1fa94387a2662773c9879206f61bfd50c76b4f0fa",
    "returncode": 2
}