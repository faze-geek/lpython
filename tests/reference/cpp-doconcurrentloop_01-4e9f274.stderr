semantic error: 'f32[:]' object does not support item assignment
 --> tests/doconcurrentloop_01.py:8:9
  |
8 |         c[i] = a[i] + scalar * b[i]
  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
